Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Main {\r\n    public static void main(String[] args){\r\n        System.out.println(\"Hey\");\r\n        System.out.println(\"Hey from bobert\");\r\n\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java	(revision aa41a399a49c24869d3e4bb30d21791b96af8a01)
+++ b/src/Main.java	(date 1636669428705)
@@ -1,7 +1,5 @@
 public class Main {
     public static void main(String[] args){
         System.out.println("Hey");
-        System.out.println("Hey from bobert");
-
     }
 }
